core/runtime.rs

```diff
@@ -1456,6 +1463,7 @@ impl JsRuntimeImpl<true> {
   ///
   /// `Error` can usually be downcast to `JsError`.
   pub fn snapshot(mut self) -> v8::StartupData {
+    // TODO(mmastrac): Why are we doing this?
     self.inner.state.borrow_mut().inspector.take();
```

Comment:
I did this as a precaution - because the isolate is getting destroyed when this function returns I didn't want to retain and reference to the inspector